components/org.wso2.carbon.identity.captcha/src/main/java/org/wso2/carbon/identity/captcha/util/CaptchaUtil.java

```diff
@@ -418,6 +436,18 @@ private static void setReCaptchaConfigs(Properties properties) {
         }
         CaptchaDataHolder.getInstance().setReCaptchaRequestWrapUrls(reCaptchaRequestWrapUrls);
 
+        String reCaptchaScoreThreshold = properties.getProperty(CaptchaConstants.RE_CAPTCHA_SCORE_THRESHOLD);
+        if (reCaptchaScoreThreshold == null) {
+            throw new RuntimeException(getValidationErrorMessage(CaptchaConstants.RE_CAPTCHA_SCORE_THRESHOLD));
+        }
+        try {
+            CaptchaDataHolder.getInstance().setReCaptchaScoreThreshold(Double.parseDouble(reCaptchaScoreThreshold));
+        } catch (NumberFormatException e) {
+            if (log.isDebugEnabled()) {
+                log.info("Error parsing recaptcha.threshold from config. Hence using the default value : 0.5", e);
```

Comment:
log.debug

Reply:
have a varaible with default value 0.5 and use it here

Reply:
Added to CaptchaUtil as
`// default value for threshold for score in reCAPTCHA v3
    private static final double CAPTCHA_V3_THRESHOLD = 0.5;`